NickvisionTagger.GNOME/Views/MainWindow.cs

```diff
@@ -1024,10 +1029,8 @@ private bool SelectedMusicFilesPropertiesChanged()
             }
             else
             {
-                using var loader = GdkPixbuf.PixbufLoader.New();
-                gdk_pixbuf_loader_write(loader.Handle, art.Data, (uint)art.Data.Length, 0);
-                _albumArtImage.SetPixbuf(loader.GetPixbuf());
-                gdk_pixbuf_loader_close(loader.Handle, IntPtr.Zero);
+                var texture = gdk_texture_new_from_bytes(g_bytes_new(art.Data, (uint)art.Data.Length), IntPtr.Zero);
```

Comment:
g_object_unref needs to he called on gdk.texture and g.bytes after the picture is set

Reply:
GBytes are not GObject, using unref on them causes silent crash.
Added unref on texture.

Reply:
GBytes has g_bytes_unref function to use